Description: Support for spaces in filenames
 This patch adds support for spaces in filenames
Author: Mazen NEIFER <mazen.neifer@wanadoo.fr>
Bug-Debian: http://bugs.debian.org/234879

--- latex2html-2008-debian1.orig/latex2html.pin
+++ latex2html-2008-debian1/latex2html.pin
@@ -829,11 +829,11 @@ sub driver {
 #	die "File $TEXEXPAND does not exist or is not executable\n"
 #	    unless (-x $TEXEXPAND);
 	L2hos->syswait("$TEXEXPAND $dbg -auto_exclude $unseg"
-		 . "-save_styles $DESTDIR$dd$TMP_${dd}styles "
-		 . ($TEXINPUTS ? "-texinputs $TEXINPUTS " : '' )
+		 . "-save_styles \"$DESTDIR$dd$TMP_${dd}styles\" "
+		 . ($TEXINPUTS ? "-texinputs \"$TEXINPUTS\" " : '' )
 		 . (($VERBOSITY >2) ? "-verbose " : '' )
-		 . "-out $DESTDIR$dd$TMP_$dd$FILE "
-		 . "$texfilepath$dd$FILE.$EXT")
+		 . "-out \"$DESTDIR$dd$TMP_$dd$FILE\" "
+		 . "\"$texfilepath$dd$FILE.$EXT\"")
 	    && die " texexpand  failed: $!\n";
 	print STDOUT "\n ***  `texexpand' done ***\n" if ($VERBOSITY > 1);
 
